app/shared/resource-daily-report-button/ResourceDailyReportButton.js

```diff
@@ -0,0 +1,26 @@
+import moment from 'moment';
+import React, { PropTypes } from 'react';
+import Button from 'react-bootstrap/lib/Button';
+
+import { buildAPIUrl } from 'api/actions/createApiAction';
+
+ResourceDailyReportButton.propTypes = {
+  date: PropTypes.string.isRequired,
+  resourceIds: PropTypes.arrayOf(PropTypes.string).isRequired,
+};
+
+function ResourceDailyReportButton({ date, resourceIds }) {
+  function getReportLink() {
+    return buildAPIUrl('reports/day_events', {
```

Comment:
- [ ] The endpoint name was changed after all. It's going to be `reports/daily_reservations`.